src/main/controllers/ManualUploadController.ts

```diff
@@ -41,6 +41,9 @@ export default class ManualUploadController {
       req.body['court'] = await manualUploadService.appendCourtId(req.body['input-autocomplete']);
       req.body['artefactType'] = 'LIST'; //Agreed on defaulting to only option available until more types become ready
       req.body['fileName'] = req.file['originalname'];
+      req.body['display-from'] = manualUploadService.joinDateValues(req.body, 'display-date-from');
```

Comment:
theres already this method in the manual upload servivce called `buildDate`

Reply:
didn't see that, removing mine